static/js/main.js

```diff
@@ -57,6 +57,8 @@ function init() {
             CURRENT_CASE = k;
             CURRENT_CASENAME = cases[k].display_name;
             $('#currentCaseDisplayName #name').text(cases[k].display_name);
+            $('#gender').addClass(cases[k].gender == 'Non-binary' ? 'ion-transgender' : 'ion-'+cases[k].gender.toLowerCase());
```

Comment:
Let's break these long strings into more readable chunks. Don't have to complete in this PR; I'll integrate it with auto-linting soon.